styles/citationEditor.less

```diff
@@ -203,7 +203,7 @@
 	line-height: 0;
 	font-size: 0;
 	border: solid 0px;
-	background: #cbe1fb url({$baseUrl}/lib/pkp/styles/splitter/ui-bg_pane.gif) 0% 50%;
+	background: #cbe1fb url(@{base-url}/lib/pkp/styles/splitter/ui-bg_pane.gif) 0% 50%;
```

Comment:
Got it fixed. Although there's no file at `/lib/pkp/styles/splitter/ui-bg_pane.gif`. There are lots of phantom styles around like this. Hard to know what can be removed or not...